notebooks/MAST/downloading_transiting_planet_data/downloading_exoplanet_data.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>I'm not sure what "the exoplanets/identifiers table" is. Is it like a lookup table? I assume it can "decode" the name as I've written it into some standardized identifier (especially peeking ahead a bit). So like if I wrote "WASP12b", would I be able to get to the same place? Okay, I download the NB myself and tried it and the answer is yes :-) I also noticed that if I use one of the other names that appears in the dictionary created below, e.g. "2MASS J06303279+2940202 b", I also get to the same entry with the same canonical name 'WASP-12 b'.</p><p> </p><p>Some explanation here about the role (and power!) of the "identifiers" table would be good for someone new to exoplanets. It was reassuring to me to discover that I could use basically any of the catalog names, or make slight variations to the string, and still get to the right entry.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>